flycheck.el

```diff
@@ -6920,6 +6921,24 @@ See URL `http://www.erlang.org/'."
    (error line-start (file-name) ":" line ": " (message) line-end))
   :modes erlang-mode)
 
+
+(flycheck-define-checker erlang-rebar3
+  "An Erlang syntax checker using the rebar3 build tool.
+
+See URL `https://www.rebar3.org'."
+  :command ("rebar3" "compile")
```

Comment:
I presume the syntax checker compiles the entire project? The it needs a :predicate that checks whether the current buffer is saved because in this case Flycheck won't be able to feed the actual contents of the buffer to the checker, so if the buffer is not saved the errors will not be accurate.